app/models/embed/purl/resource.rb

```diff
@@ -51,6 +51,11 @@ def thumbnail
       def caption_files
         files.select(&:caption?).sort_by(&:language_code)
       end
+
+      # @return [Array<ResourceFile>]
+      def transcript_files
```

Comment:
Is this method used anywhere?

Reply:
@jcoyne: Nope - that was part of my first and will be removed before I undraft.